NUTranslator/status.js

```diff
@@ -0,0 +1,17 @@
+import { getLogger } from '../Logger';
+
+const options = {
+  '0': 'Good',
```

Comment:
we are using iN18 to translate all the strings we need to be able to translate these strings as well! Please open an issue for now and will look into this later